scipy/stats/_mstats_basic.py

```diff
@@ -2564,10 +2564,13 @@ def moment(a, moment=1, axis=0):
     axis : int or None, optional
        Axis along which the central moment is computed. Default is 0.
        If None, compute over the whole array `a`.
+    center : float or None, optional
+       The point about which moments are taken. This can be the sample mean, the
+       origin, or any other be point. If None, compute from the sample.
```

Comment:
```suggestion
       The point about which moments are taken. This can be the sample mean,
       the origin, or any other point. If None (default), use the mean
       computed from the sample.
```
Need to specify what is computed from the sample. Also, we need to specify that computing the mean from the sample is the default.